src/toolbar/toolbar-group.jsx

```diff
@@ -197,7 +196,7 @@ const ToolbarGroup = React.createClass({
     }, this);
 
     return (
-      <div {...other} className={className} style={this.prepareStyles(styles.root, style)}>
+      <div {...other} className={className} style={prepareStyles(Object.assign({}, styles.root, style))}>
```

Comment:
That's fair. I see a performance concern in this approach. 50%+ of the time, the parameter passed to the `prepareStyles` is safe to mutate. I feel like we will end up with more `Object.assign({},` with your solution.